### ISSUE TITLE
Icons are blank

### ISSUE THREAD

**Issue by redworkerbee** [grokability/snipe-it#11257]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I did a fresh install of snipe it this morning. All the icons are replaced by squares and I'm not sure where I went wrong. Do I need to roll back something to a previous version? I have tried opening in chrome and firefox but both are the same.

snipe it: 6.0.2
php: 8.1.6
mariadb: 10.6.8
php manager: 2.7

### Reproduction steps

1. Open snipe it
2.
3.
...

### Expected behavior

able to see icons

### Screenshots

![Screenshot 2022-06-06 112440](https://user-images.githubusercontent.com/106985597/172194803-47a00cc1-62f6-4487-ab8c-c9c5137b3538.png)
![Screenshot 2022-06-06 112504](https://user-images.githubusercontent.com/106985597/172194811-cc758e50-0547-4d9d-978e-1d3929adb26e.png)
![Screenshot 2022-06-06 113557](https://user-images.githubusercontent.com/106985597/172194825-5428a0d7-3486-4b7f-bda7-fa3fadf353eb.png)

### Snipe-IT Version

6.0.2

### Operating System

Windows Server 2022

### Web Server

IIS

### PHP Version

8.1.6

**Comment by snipe** [comment]
Are you seeing anything in your browser's console? (Errors, etc)

**Comment by snipe** [comment]
(Also can you reproduce this on the demo? https://demo.snipeitapp.com)

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>

**Comment by redworkerbee** [comment]
I don't see any errors in the browser. All the links work fine, but for some reason it's just the issue with the icons.

**Comment by triple-HA** [comment]
Its looks like folder privilages.... What if you click on those square and choose "open graphic in new tab" ?

**Comment by redworkerbee** [comment]
I have checked the folder permissions and IUSR has been given full permissions to cache, storage, upload folders. modify for the temp folder. 
That option doesn't pop up.

**Comment by triple-HA** [comment]
Ok, ive checked that and it looks like you dont have font awesome....

**Comment by redworkerbee** [comment]
Yea, looks like it's not there. Where can I get it? Is it supposed to come with the 6.0.2 install?

**Comment by triple-HA** [comment]
I could check this installation on my environment tomorrow. Could you send me install instruction what you use?
Did you check your installation on localhost and outside?

**Comment by snipe** [comment]
The fonts are included in the repo

**Comment by redworkerbee** [comment]
Ok I found it under build.

**Comment by redworkerbee** [comment]
Some icons show up under the People tab.

I followed this video: https://www.youtube.com/watch?v=kxGGQutvktM

**Comment by snipe** [comment]
You shouldn't need to move anything though - they're in the build directory, and that's where they pull from. I'm kinda confused as to what's going on here.

**Comment by redworkerbee** [comment]
I didn't move any files. I just don't know why the icons are not showing up. I also tried restoring a backup, and although it said it was a success, it doesn't show up. I also can't seem to import anything either. So I'm not sure there's a config I messed up somewhere or if it's another issue.

I had installed & configured snipe it before with a previous version, and everything had worked correctly. I'm not sure where I messed up this time.
